tests/components/create_post/create_post.test.jsx

```diff
@@ -629,6 +631,27 @@ describe('components/create_post', () => {
         expect(onSubmitPost.mock.calls[0][1]).toEqual([]);
     });
 
+    it('Should have called actions.selectPostFromRightHandSideSearchByPostId on replyToLastPost', () => {
+        const selectPostFromRightHandSideSearchByPostId = jest.fn();
+        let latestReplyablePostId = '';
+        const wrapper = shallow(createPost({
+            actions: {
+                ...actionsProp,
+                selectPostFromRightHandSideSearchByPostId
+            },
+            latestReplyablePostId
+        }));
+
+        wrapper.instance().replyToLastPost({preventDefault: jest.fn()});
+        expect(selectPostFromRightHandSideSearchByPostId).not.toBeCalled();
+
+        latestReplyablePostId = 'latest_replyablePost_id';
+        wrapper.setProps({latestReplyablePostId});
+        wrapper.instance().replyToLastPost({preventDefault: jest.fn()});
+        expect(selectPostFromRightHandSideSearchByPostId).toHaveBeenCalledTimes(1);
+        expect(selectPostFromRightHandSideSearchByPostId.mock.calls[0][0]).toEqual(latestReplyablePostId);
+    });
+
```

Comment:
This is just an additional test case to CreatePost which is not related to this PR.